scripts/new_project.kts

```diff
@@ -61,6 +63,23 @@ object NewProject {
         val process = Runtime.getRuntime().exec(command)
         process.inputStream.reader().forEachLine { println(it) }
     }
+
+    private fun renamePackageNames() {
+        showMessage("=> 🔎 Renaming package name within files...")
+        File("../$appNameWithoutSpace")?.walk().forEach {
+            val filePath = it.toString()
+            rename(sourcePath = filePath, keyword = OLD_PACKAGE_NAME, changeTo = packageName)
+        }
+    }
+
+    private fun rename(sourcePath: String, keyword: String, changeTo: String) {
+        if (File(sourcePath)?.isDirectory.not()) {
```

Comment:
Instead of doing the check here, would it be possible to filter it beforehand? 🎸 

Ex: 
```
File("../$appNameWithoutSpace")
    ?.walk()
    .filter { it.isDirectory.not() }
    .forEach { ... }
```


If I'm not mistaken, it will be more performant this way